cmd/helms3/main.go

```diff
@@ -24,6 +24,9 @@ const (
 	defaultTimeout       = time.Minute * 5
 	defaultTimeoutString = "5m"
 
+	// duplicated in e2e package for testing.
+	defaultChartsContentType = "application/x-gzip"
```

Comment:
Should we use `application/gzip` instead, as defined per [RFC 6713](https://tools.ietf.org/html/rfc6713)? Also: https://www.iana.org/assignments/media-types/media-types.xhtml

Reply:
Yes, x-gzip is deprecated.